config/routes.rb

```diff
@@ -2,4 +2,7 @@
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
 
   resources :posts
+  # get  '/signup',  to: 'users#new'
+  resources :users
+  root 'users#new'
```

Comment:
Does this line set the homepage root url to the content from /users/new?

Reply:
yes it does

Reply:
Cool